keyboards/planck/keymaps/ishtob/keymap.c

```diff
@@ -331,12 +230,12 @@ void startup_user()
     rgblight_mode(RGB_MATRIX_CYCLE_ALL);
     #endif //RGB_matrix
     wait_ms(20); // gets rid of tick
-    PLAY_NOTE_ARRAY(tone_startup, false, 0);
+    PLAY_SONG(tone_startup);
```

Comment:
Isn't needed if you define this in the config.h:
https://docs.qmk.fm/#/feature_audio

Reply:
removed